main.nf

```diff
@@ -2582,11 +3261,26 @@ process ControlFreecViz {
     when: 'controlfreec' in tools
 
     """
-    cat /opt/conda/envs/nf-core-sarek-${workflow.manifest.version}/bin/assess_significance.R | R --slave --args ${cnvTumor} ${ratioTumor}
-    cat /opt/conda/envs/nf-core-sarek-${workflow.manifest.version}/bin/assess_significance.R | R --slave --args ${cnvNormal} ${ratioNormal}
+    echo "Shaping CNV files to make sure we can assess significance"
+    awk 'NF==9{print}' ${cnvTumor} > TUMOR.CNVs
+    awk 'NF==7{print}' ${cnvNormal} > NORMAL.CNVs
+
+    echo "############### Calculating significance values for TUMOR CNVs #############"
+    cat /opt/conda/envs/nf-core-sarek-${workflow.manifest.version}/bin/assess_significance.R | R --slave --args TUMOR.CNVs ${ratioTumor}
```

Comment:
I'm not sure why here is needed to call the script like this. Wouldn't it be possible to just call `assess_singificance.R` like the Ascat R scripts? It just needs the right shebang line.

Reply:
And to be added to `bin/` as now I realize it is not there

Reply:
Can't be added to `bin/` as it's part of the conda environment.
Tried to call it the same way I did with the other script, but it wasn't working.
This is how they do it in the Control-FREEC docs

Reply:
ah, ok, so it comes delivered inside the container with some of the tools? In that case that won't work, right

Reply:
I guess the recipe for the conda could be improved so that it's in the `PATH` and can be called directly.
I'll think about that for the next release, there are other recipes that need some love too